apps/wallet/src/shared/utils/bip39.ts

```diff
@@ -1,9 +1,8 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
-import {
-    generateMnemonic as bip39GenerateMnemonic,
-    validateMnemonic as bip39ValidateMnemonic,
-} from '@scure/bip39';
+import { bytes as assertBytes } from '@noble/hashes/_assert';
```

Comment:
This feels like an internal import we shouldn't be doing. This function seems small enough that we can just port it over here.